osf/models/registrations.py

```diff
@@ -74,6 +74,7 @@ class Registration(AbstractNode):
                                                     related_name='registrations',
                                                     null=True, blank=True,
                                                     on_delete=models.SET_NULL)
+    files_count = models.PositiveIntegerField(blank=True, null=True)
```

Comment:
hmm, should we make the default 0 instead of None?

Reply:
I think `None` is good, so we can tell whether it's actually been counted. Feels right to just default to 0 in the API, as it does 👍